docs/_docs/installation/macos.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Jekyll on macOS
+permalink: /docs/installation/macos/
+---
+
+First, you need to install the command-line tools to be able to compile native extensions, open a terminal and run:
+
+```sh
+xcode-select --install
+```
+
+## Set up Ruby included with the OS
+
+Check your Ruby version meet our requirements. Jekyll requires Ruby 2.2.5 or above. If you're running an older version you'll need to [install a more recent Ruby version via Homebrew](#homebrew).
+
+```sh
+ruby -v
+2.3.3
+```
+
+Now install Jekyll and [Bundler](/docs/ruby-101/#bundler).
+
+```sh
+gem install bundler jekyll
+```
+
+### Install a newer Ruby version via Homebrew {#homebrew}
+
+If you wish to install the latest version of Ruby and get faster builds, we recommend to do it via [Homebrew](https://brew.sh) a handy package manager for macOS.
+
+```sh
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+brew install ruby
+ruby -v
+ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-darwin17]
+```
+
+### Install multiple Ruby versions with rbenv {#rbenv}
+
+Developers often use [rbenv](https://github.com/rbenv/rbenv) to manage multiple Ruby versions. This can be useful if you want to run the same Ruby version used by 3rd party hosting providers.
```

Comment:
"used by 3rd party hosting providers" isn't _that_ convincing an argument, i'd say something like "used by your colleagues/collaborators" is more realistic